### PR TITLE
ADD: test workflow

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
```
Comment (iTitus) [inline .github/workflows/python-app.yml:9]: this might need a branch selector to work

Comment (Segelzwerg) [reply .github/workflows/python-app.yml:9]: I copy-pasted from here and it worked there: https://github.com/Segelzwerg/FamilyFoto/blob/master/.github/workflows/python-check.yml

Comment (iTitus) [reply .github/workflows/python-app.yml:9]: So currently it will only run on pushes to main, but on PR to all branches?

Comment (Segelzwerg) [reply .github/workflows/python-app.yml:9]: Exactly

**Review by iTitus — APPROVED** [review]

**Pull request merged**